agent/agent.go

```diff
@@ -226,6 +246,87 @@ func (a *Agent) initPlugins() error {
 	return nil
 }
 
+// initPersister initializes the persister and registers the plugins.
+func (a *Agent) initPersister() error {
+	if err := a.Config.Persister.Init(); err != nil {
+		return err
+	}
+
+	for _, input := range a.Config.Inputs {
+		plugin, ok := input.Input.(telegraf.StatefulPlugin)
+		if !ok {
+			log.Printf("D! [agent] Skipping registration of non-stateful plugin %q...", input.LogName())
```

Comment:
Won't this give a lot of useless debug messages? One for each plugin/instance..

Reply:
Well it's only once during startup and any in debug mode... So I guess that's ok?

Reply:
In my dev setup it would introduce around 3000 log lines that are actually saying nothing interesting; We will know by the first line that `inputs.ping` is not stateful, no need to repeat that the other 1800 times..

Reply:
You are right. Will remove that output.